QuizApp/Core/Custom UI/FinishPopup/FinishPopupController.swift

```diff
@@ -0,0 +1,199 @@
+//
+//  FinishPopup.swift
+//  QuizApp
+//
+//  Created by Giorgi Makadze on 09.06.2023.
+//
+
+import Foundation
+import UIKit
+
+class FinishPopupController: UIViewController {
+    //MARK: -  Components
+    
+    private lazy var popupView: UIView = {
+        let popupView = UIView()
+        popupView.backgroundColor = Constants.PopupView.backgroundColor
+        popupView.layer.cornerRadius = Constants.PopupView.cornerRadius
+        popupView.translatesAutoresizingMaskIntoConstraints = false
+        view.addSubview(popupView)
+        return popupView
+    }()
+    
+    private lazy var successIcon: UIImageView = {
+        let successIcon = UIImageView()
+        successIcon.image = Constants.SuccessIcon.image
+        successIcon.contentMode = .scaleAspectFill
+        successIcon.translatesAutoresizingMaskIntoConstraints = false
+        popupView.addSubview(successIcon)
+        return successIcon
+    }()
+    
+    private lazy var popupLabel: UILabel = {
+        let popupLabel = UILabel()
+        popupLabel.text = Constants.PopupLabel.text
+        popupLabel.backgroundColor = Constants.PopupLabel.backgroundColor
+        popupLabel.font = Constants.PopupLabel.font
+        popupLabel.textColor = Constants.PopupLabel.textColor
+        popupLabel.translatesAutoresizingMaskIntoConstraints = false
+        popupView.addSubview(popupLabel)
+        return popupLabel
+    }()
+    
+    private lazy var popupScoreLabel: UILabel = {
+        let popupScoreLabel = UILabel()
+        popupScoreLabel.text = Constants.PopupScoreLabel.text
+        popupScoreLabel.backgroundColor = Constants.PopupScoreLabel.backgroundColor
+        popupScoreLabel.font = Constants.PopupScoreLabel.font
+        popupScoreLabel.textColor = Constants.PopupScoreLabel.textColor
+        popupScoreLabel.translatesAutoresizingMaskIntoConstraints = false
+        popupView.addSubview(popupScoreLabel)
+        return popupScoreLabel
+    }()
+    
+    private lazy var dividerView: UIView = {
+        let dividerView = UIView()
+        dividerView.backgroundColor = Constants.DividerView.backgroundColor
+        dividerView.translatesAutoresizingMaskIntoConstraints = false
+        popupView.addSubview(dividerView)
+        return dividerView
+    }()
+    
+    private lazy var closeButton: UIButton = {
+        let closeButton = UIButton()
+        closeButton.backgroundColor = Constants.CloseButton.backgroundColor
+        closeButton.setTitle(Constants.CloseButton.setTitle, for: .normal)
```

Comment:
შეგიძია არ შეცვალო, მაგრამ title უკეთესი სახელი იქნებოდა
```suggestion
        closeButton.setTitle(Constants.CloseButton.title, for: .normal)
```